[WIP] Enterprise 24.1 release notes (#75)
seqeralabs/docs: platform_versioned_docs/version-24.1.0/data/data-explorer.mdx

---
title: "Data Explorer"
description: "Using Seqera Data Explorer."
date: "21 Apr 2023"
tags: [data, explorer]
---

With Data Explorer, you can browse and interact with remote data repositories from organization workspaces in Seqera Platform. It supports AWS S3, Azure Blob Storage, and Google Cloud Storage repositories.

:::note
To disable Data Explorer globally for all workspaces and users in your account, set `TOWER_DATA_EXPLORER_ENABLED=false` in your environment variables (default: `true`).

> To disable Data Explorer automatic cloud bucket retrieval per workspace in your **Seqera Enterprise** account, specify comma-separated workspace IDs with the `TOWER_DATA_EXPLORER_CLOUD_DISABLED_WORKSPACES` environment variable.

Reviewer: Fixing.

  :::note
  The **Add cloud bucket** option is only shown to users with sufficient permissions (workspace maintainers or higher).
  :::

## Browse cloud storage buckets

![](./_images/data_explorer.png)

- **View bucket details**

  To view bucket details such as the cloud provider, bucket address, and credentials, select the information icon next to a bucket in the Data Explorer list.

- **Search and filter buckets**

  Search for buckets by name and region (e.g., `region:eu-west-2`) in the search field, and filter by provider.

- **Hide buckets from list view**

  Workspace maintainers can hide buckets from the Data Explorer list view. Select multiple buckets, then select **Hide** in the Data Explorer toolbar. To hide buckets individually, select **Hide** from the options menu of a bucket in the list.

  The Data Explorer list filter defaults to **Only visible**. Select **Only hidden** or **All** from the filtering menu to view hidden buckets in the list. You can **Unhide** a bucket from its options menu in the list view.

- **View bucket contents**

>   Select a bucket name from the Data Explorer list to view the contents of that bucket. From the **View cloud bucket** page, you can browse directories and search for objects by name in a particular directory. The file type, size, and path of objects are displayed in columns to the right of the object name. To view bucket details such as the cloud provider, bucket address, and credentials, select the information icon.

Reviewer 2: File type/kind column was [recently removed](https://github.com/seqeralabs/platform/pull/6728). Update:

```suggestion
  Select a bucket name from the Data Explorer list to view the contents of that bucket. From the **View cloud bucket** page, you can browse directories and search for objects by name in a particular directory. The size and path of an object is displayed in columns to the right of the object name. To view bucket details such as the cloud provider, bucket address, and credentials, select the information icon.
```
